### PR TITLE
Added Project files and MyLinkedList implementation.

### PR DESCRIPTION
**Pull request by xenuvix**
Additionally fixed StoreQueueTest.java

### CODE REVIEW

**Comment by DanielLavie** [comment]
Do we really need all the intellij generated files commited?

File: .idea/compiler.xml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (DanielLavie) [inline .idea/compiler.xml]: Do we really need al the intellij generated files commited?

File: Node.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    /** Constructs a node with the given data.
+     * The new node will point to null.
+     */
+    Node(T data) {
+        this(data, null);
+    }
+    
+    /** Returns the data of this node, as a string.
+     */
+    public String toString() {
```
Comment (DanielLavie) [inline Node.java]: Person dont have a toString() method implemented, do we need this method?

**Comment by xenuvix** [comment]
We don’t, I thought you could use them as well. Anyway we’re not sending them to the examiners.

File: Node.java (hunk 2 above)
Comment (xenuvix) [inline Node.java]: We don’t really. I deleted it.

**Pull request merged**